Sources/SwiftSyntax/ArenaAllocatedBuffer.swift

```diff
@@ -11,22 +11,22 @@
 //===----------------------------------------------------------------------===//
 
 /// An `UnsafePointer` pointing into memory that was allocated by a
-/// ``SyntaxArena``.
+/// ``RawSyntaxArena``.
 ///
 /// Because the syntax arena will always outlive any syntax nodes that
 /// reference its contents, we know that the pointer's contents won't get
 /// deallocated while being accessed and thus we can add an unchecked `Sendable`
 /// conformance.
 @_spi(RawSyntax)
-public struct SyntaxArenaAllocatedPointer<Element: Sendable>: @unchecked Sendable {
+public struct ArenaAllocatedPointer<Element: Sendable>: @unchecked Sendable {
```

Comment:
Should the comment also be updated so it doesn’t mention `RawSyntaxArena` if this type is supposed to hold pointers allocated in arbitrary arenas?

Same for comments below.